disabled-tests-inspector/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.quarkus.ts</groupId>
+    <artifactId>disabled-tests</artifactId>
```

Comment:
Why does Disabled tests inspector in `disabled-tests-inspector` has artifact id `disabled-tests`?